pypeit/reduce.py

```diff
@@ -371,7 +369,7 @@ def run_objfind(self, std_trace=None, show_peaks=False):
         # First pass object finding
         self.sobjs_obj, self.nobj, skymask_init = \
             self.find_objects(self.sciImg.image, std_trace=std_trace,
-                              show_peaks=show_peaks,
+                              show_peaks=self.par['reduce']['findobj']['skip_second_find'],
```

Comment:
Let's discuss these options. I think we want to have the option to have interactive plotting of both object finding steps since that is useful for debugging. Currently you are disabling that. This does not exclude the option to write out the result of the second object finding to a QA file, and so we should also have the option to do that.